index.js

```diff
@@ -0,0 +1,377 @@
+const express = require('express'); // pull in express
+const cors = require('cors');
+const server = express(); // instantiate server
+const logger = require('morgan');
+const db = require('./data/helpers/userDb');
+const postDb = require('./data/helpers/postDb');
+const port = 5201;
+
+
+const capitalize = (req, res, next) => {
+  const { name } = req.body;
+  req.body.name = req.body.name.charAt(0).toUpperCase() + req.body.name.slice(1);
+  next();
+};
+
+const textRecognize = (req, res, next) => {
+  const newText = req.body.text;
+  req.text = newText
+};
+
+
+server.use(cors()); // connects react
+server.use(express.json()); 
+server.use(logger('combined'));
+
+
+
+server.listen(port, () => {
+  console.log(`================Running on port ${port}==============`);
+});
+
+
+
+
+/*                        USERS                              */
+
+server.get('/', (req, res) => {
+  res.send('<h3>Welcome to INSTANT-GRAM!</h3>');
+});
+
+server.get('/users', (req, res) => {
+  db.get()
+    .then(users => {
+      console.log(' I am users name: ', users);
+      res.json(users);
+    })
+    .catch(err => res.send(err))
+});
+
+
+
+
+
+
+
+
+server.post('/users', capitalize, (req, res) => {    
+
+  db.insert(req.body)
+    .then(user => {
+      // res.status(201).json(user);
+      db.get(user.id)
+        .then(userName => {
+          if (!req.body.name) {
+            res.status(404).json({ missingError: 'Provide a name' });
```

Comment:
404 means not found or does not exist. In this case, we're just sending invalid data, so 400 would be better.